data/pc/1.19/protocol.json

```diff
@@ -5602,7 +5608,7 @@
           [
             {
               "name": "slotId",
-              "type": "i8"
```

Comment:
whats the point in this change, it's not valid to send a slot id > 8, as the hotbar is 9 slots long...

Reply:
Yeah, this is serializing into a stream of bits. Writing wrong size will mean the number will be read incorrectly (especially since it's big endian not little), as will further fields be in the wrong position.

Reply:
Okay sounds right